README.md

```diff
@@ -61,6 +61,16 @@ All configurations are simple YAML files which you can copy, share, or edit manu
 
 After you save changes, if you have already started handling, you need to click "Restart handling" to start a new server with the changes applied.
 
+## Usage without GUI
+
+The library can be also controlled using the singleton of [`StateManager`](api_reference.md) class:
```

Comment:
Does anyone know how to add a direct reference to the `StateManager` section in the API reference?